dev/deps/spark-deps-hadoop-2-hive-2.3

```diff
@@ -228,7 +232,7 @@ scala-xml_2.12/1.2.0//scala-xml_2.12-1.2.0.jar
 shapeless_2.12/2.3.3//shapeless_2.12-2.3.3.jar
 shims/0.9.22//shims-0.9.22.jar
 slf4j-api/1.7.30//slf4j-api-1.7.30.jar
-slf4j-log4j12/1.7.30//slf4j-log4j12-1.7.30.jar
+slf4j-log4j12/1.7.10//slf4j-log4j12-1.7.10.jar
```

Comment:
Do we need to rollback this dependency? Could you keep this at 1.7.30 in `hadoop2` profile?